recipe/meta.yaml

```diff
@@ -12,13 +12,8 @@ source:
 build:
   number: 0
   #skip s390x as lack of nodejs and cartopy dependencies
-  skip: True  # [py<39 or s390x]
-requirements:
-  host:
-    - python
-    - pip
-    - setuptools
-    - wheel
```

Comment:
@ahuang11 Could you revert this change? Platforms will fail on our CI (Prefect) randomly if `meta.yaml` doesn't have a generic r`equirements/host` section

Reply:
Is there a conda-build issue filed for this? Is it because of skip py on the global level without python being on the global env?